packages/airnode-utilities/src/caching/index.ts

```diff
@@ -133,51 +143,55 @@ const addKey = (key: string, data: any, force = false) => {
  * Sweep (garbage collect) the cache.
  */
 const sweep = () => {
-  try {
-    const now = Date.now();
+  const now = Date.now();
 
-    const stattedFiles = readdirSync(CACHE_BASE_PATH).map((file) => ({
+  const goRes = goSync(() =>
```

Comment:
This now changed the behaviour. Previously all of the code was inside the try-catch. 

EDIT: I see the rest of the code does not throw so it's probably fine.